barretenberg/cpp/src/barretenberg/ultra_honk/ultra_transcript.test.cpp

```diff
@@ -42,7 +42,7 @@ class UltraTranscriptTests : public ::testing::Test {
         size_t frs_per_uint32 = bb::field_conversion::calc_num_bn254_frs<uint32_t>();
 
         size_t round = 0;
-        manifest_expected.add_entry(round, "circuit_size", frs_per_uint32);
+        manifest_expected.add_entry(round, "instance_size", frs_per_uint32);
```

Comment:
hacky-ish fix but seems like the best thing to do to align ultra honk prover and folding prover